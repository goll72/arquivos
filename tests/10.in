1 ataque10.csv ataque10.bin