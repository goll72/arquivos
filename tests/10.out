174413.510000
